src/client/Client.cc

```diff
@@ -12831,13 +12902,8 @@ size_t Client::_vxattrcb_client_id(Inode *in, char *val, size_t size)
 }
 
 const Client::VXattr Client::_dir_vxattrs[] = {
-  {
-    name: "ceph.dir.layout",
-    getxattr_cb: &Client::_vxattrcb_layout,
-    readonly: false,
-    exists_cb: &Client::_vxattrcb_layout_exists,
-    flags: 0,
-  },
+  // FIXME
+  // Delete the following dir layout field definitions for release "S"
   XATTR_LAYOUT_FIELD(dir, layout, stripe_unit),
   XATTR_LAYOUT_FIELD(dir, layout, stripe_count),
   XATTR_LAYOUT_FIELD(dir, layout, object_size),
```

Comment:
@batrick Can't we just delete old style dir and file layout vxattr processing from the fuse client since we now have both the old and new way of handling these vxattr values at the server side via this PR?

Reply:
No, then the client wouldn't handle layout xattrs for older Ceph clusters.